cadastre/cadastre_common_base.py

```diff
@@ -361,7 +361,7 @@ def getCompteCommunalFromParcelleId(parcelleId: str, connectionParams: Dict[str,
 
 
 def getProprietaireComptesCommunaux(comptecommunal: str, connectionParams: Dict[str, str],
-                                    connector: 'DBConnector') -> List[str]:
+                                    connector: 'DBConnector', allCities: bool = True) -> List[str]:
```

Comment:
I would simply return the given comptecommunal
```python
if not allCities:
    return comptecommunal
```

Or I have missed something ?

Reply:
Certains propriétaires ont plusieurs compte communaux sur 1 même commune

Reply:
Oui, et ils ont chacun un DNUPER différent normalement. J'avais pensé concaténer le code DNUPER dans le champ de recherche/résultat de l'autocomplétion, par ex:

`COMMUNE DE MACHIN - PBBBM0` , sauf si c'est pas ce qui est souhaité ?

Reply:
Pour info et pouvoir tester, le dnuper est visible dans l'onglet `Propriétaires` visible dans la popup, champ `Code`